napari_animation/animation.py

```diff
@@ -243,3 +246,10 @@ def _on_active_keyframe_changed(self, event):
         if active_keyframe:
             keyframe_index = self.key_frames.index(active_keyframe)
             self.set_key_frame_index(keyframe_index)
+
+    def _repr_html_(self):
+        if self._filename is None:
+            return 'Video animation not yet available (use the "animate" method to generate it).'
```

Comment:
This is the message you see if the animation has not yet been saved. I think it's clear, but welcome suggestions to improve it.

![Screen Shot 2023-05-01 at 5 39 21 pm](https://user-images.githubusercontent.com/30920819/235424326-c14f9382-c002-4b98-ae88-4e4923d25f02.png)